src/features/auth/pages/login/Login.native.test.tsx

```diff
@@ -289,7 +289,7 @@ describe('<Login/>', () => {
         .mockReturnValueOnce({ params: { offerId: OFFER_ID } }) // password input rerender
     })
 
-    it('should redirect to the previous offer page when signin is successful', async () => {
+    it('should redirect to Offer page when signin is successful and user comes from offer', async () => {
```

Comment:
Le describe s'appelle `Login comes from an offer` c'est pour ça que je n'avais pas mis ` and user comes from offer` ^^